### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by phixion**
# GitHub Resume Generator

GitHub Flow PR Image

Thank you so much for opening a pull request! :smiley:

Please provide the following information where possible to help the pull request reviewer merge your PR quicker!

## Relevant Project Issue Numbers :hash:

#15

## Involved Project Members :bust_in_silhouette:

@sabesansathananthan

## An Explanation of Your Changes :speech_balloon:

Aded German translation and fixed minor markdown formatting

## For transilators.

- [x] Create a file called `README.lang.md` in the translations folder.
      lang means shorten form of your language Refer this [site](https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code).

- [x] Check all the links are working.

- [x] Check all the parts are translated.
- [x] If about and built-with are not working then inspect the page and copy the href and paste in your file.
- [x] Edit the translations/Translations.md adding you language icon/link
- [x] Edit the main README.md adding your language icon

<!-- ## Any Screenshots of Your Changes :camera:

README de

### CODE REVIEW

File: translations/README.de.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# GitHub Resume Generator
```
Comment (sabesansathananthan) [inline translations/README.de.md:1]: Is there any options in German ?

File: translations/README.de.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+4. Öffne <http://localhost:3000> in deinem Browser
+
+<h2 id='live-demo'><a href="https://react-github-resume.vercel.app/">Live Demo</a></h2>
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 Lizenz
+
+Dieses Projekt ist lizensiert unter der MIT License - [LICENSE](../LICENSE) für Details
```
Comment (sabesansathananthan) [inline translations/README.de.md:74]: Add german instead of [LICENSE]

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]

File: translations/README.de.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+   wenn Du `yarn` nutzt:
+
+   ```bash
+   yarn start
+   ```
+
+4. Öffne <http://localhost:3000> in deinem Browser
+
+<h2 id='live-demo'><a href="https://react-github-resume.vercel.app/">Live Demo</a></h2>
```
Comment (sabesansathananthan) [inline translations/README.de.md:68]: Is Live Demo is same as Live Demo in German

File: translations/README.de.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
+# GitHub Resume Generator
+
+![React GitHub Resume](../src/assets/readme/screenshot.png)
+
+## Inhalt
+
+- [Über das Projekt](#about)
+- [Werkzeuge](#built-with)
+- [Installation](#installation)
+- [Demo](#live-demo)
```
Comment (sabesansathananthan) [inline translations/README.de.md:10]: The demo is this same as german

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]

**Comment by sabesansathananthan** [comment]
@esau-morais I reviewed Could you please review it?

**Comment by sabesansathananthan** [comment]
Please fill the PR form and Edit Main README and remove the word upcoming in German

File: translations/README.de.md (hunk 3 above)
Comment (phixion) [reply translations/README.de.md:68]: yes - its not common to call it something else

File: translations/README.de.md (hunk 4 above)
Comment (phixion) [reply translations/README.de.md:10]: yes

File: translations/README.de.md (hunk 1 above)
Comment (esau-morais) [reply translations/README.de.md:1]: > Is there any options in German ?

If you see all the translations didn't change this title

Comment (phixion) [reply translations/README.de.md:1]: I wouldnt translate (Project-)Names

**Review by sabesansathananthan — APPROVED** [review]

**Comment by sabesansathananthan** [comment]
Thank you for your PR... <3

**Comment by sabesansathananthan** [comment]
Don't forget to give a star to this repository...

**Pull request merged**